cpp/test/neighbors/ann_cagra_c.cu

```diff
@@ -66,6 +80,11 @@ TEST(CagraC, BuildSearch)
   cuvsCagraIndexParamsCreate(&build_params);
   cuvsCagraBuild(res, build_params, &dataset_tensor, index);
 
+  cuvsCagraExtendParams_t extend_params;
+  cuvsCagraExtendParamsCreate(&extend_params);
+  extend_params->max_chunk_size = 100;
+  cuvsCagraExtend(res, extend_params, &additional_dataset_tensor, index);
```

Comment:
for debugging the segfault - does this still occur when called with the original dataset - like :
```suggestion
  cuvsCagraExtend(res, extend_params, &dataset_tensor, index);
```

Reply:
still segfaults unfortunately

Reply:
@ajit283 I think @benfred might be right about this being from the number of vectors in the dataset. Can we generate a matrix using the make_blobs from raft::random ? Then we can just generate a larger dataset and bypass the issue.